docs/pages/admin-guides/access-controls/guides/headless.mdx

```diff
@@ -26,7 +26,7 @@ For example:
 ## Prerequisites
 
 - A v12.2+ Teleport cluster with WebAuthn configured.
-  See the [Second Factor: WebAuthn](./webauthn.mdx) guide.
+  See the [multi-factor: WebAuthn](./webauthn.mdx) guide.
```

Comment:
```suggestion
  See [the WebAuthn guide](./webauthn.mdx).
```

I believe in the past this guide might have been called "Second Factor: WebAuthn", but it no longer has that title. Calling it "multi-factor: WebAuthn" doesn't make much sense either IMHO, "WebAuthn" should be enough.

Reply:
Changed